aws-lc-rs/tests/rsa_test.rs

```diff
@@ -332,6 +339,84 @@ generate_fips_encode_decode!(rsa3072_generate_fips_encode_decode, KeySize::Rsa30
 generate_fips_encode_decode!(rsa4096_generate_fips_encode_decode, KeySize::Rsa4096);
 generate_fips_encode_decode!(rsa8192_generate_fips_encode_decode, KeySize::Rsa8192, false);
 
+macro_rules! encryption_generate_encode_decode {
```

Comment:
How hard would it be to add interoperability tests here with ring or other rust libraries?

Reply:
It's doable, but I'm not sure if aws-lc-rs is the right level to be extensively testing that at. We can explore having a smoketest / sanity test to just validate with a quick inter-opt for keys between another library, but we are just a pass-through to AWS-LC in terms of the keys it produces, and the serialization outputs and inputs it supports.